packages/artifacts/src/projects.ts

```diff
@@ -1,8 +1,8 @@
 export enum Project {
-  SEMAPHORE_IDENTITY = 'semaphore-identity',
   POSEIDON = 'poseidon',
   // RLN = 'rln',
   SEMAPHORE = 'semaphore',
+  SEMAPHORE_IDENTITY = 'semaphore-identity',
 }
 
-export const projects = Object.values(Project)
+export const projects = Object.values(Project).sort()
```

Comment:
I think the order of the enum is kept.
Sorting here to be extra sure and to avoid having to do it client side (we can bump the dep in the web app later)